db/seeds/news.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# After seeding the database, you should see these news items on the SERP and News facet:
+# /search?affiliate=test_affiliate&query=news
+# /search/news?affiliate=test_affiliate&channel=1&query=news
+
+affiliate = Affiliate.find_by(name: 'test_affiliate')
```

Comment:
I was torn about putting all this seed data in a single "news" file, as opposed to a file per model. My thinking was to lump the full creation flow in a single file to make it easier for a developer to see all the data setup involved in news searches, without jumping through 4 different seed files to do so. Happy to discuss alternatives. This might also just be "good enough" for now.